### DISCUSSION TITLE
外层通过transform scale缩放后，会导致el-select多选tag组件中.el-select__tags的宽度计算错误

### DISCUSSION THREAD

**Discussion by Jinzm** [element-plus/element-plus#11141] [Q&A]
el-select多选tag组件在外层通过transform scale缩放后，会导致.el-select__tags的宽度计算错误，导致内部显示出现问题
![image](https://user-images.githubusercontent.com/21049455/210482608-be86b792-1de8-46a0-af79-ba88ba205e42.png)

[playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2IGNsYXNzPVwicGxheS1jb250YWluZXJcIiBzdHlsZT1cInRyYW5zZm9ybTogc2NhbGUoMC44MjY1NiwgMC44MjY1NilcIj5cbiAgICA8ZWwtc2VsZWN0XG4gICAgICB2LW1vZGVsPVwibW9kZWxWYWx1ZVwiXG4gICAgICBzdHlsZT1cIndpZHRoOjE1MHB4XCJcbiAgICAgIDptdWx0aXBsZT1cInRydWVcIlxuICAgICAgY29sbGFwc2UtdGFnc1xuICAgICAgY29sbGFwc2UtdGFncy10b29sdGlwXG4gICAgPlxuICAgICAgPGVsLW9wdGlvbiB2YWx1ZT1cIjFcIiBsYWJlbD1cIuWFqOmDqDFcIj48L2VsLW9wdGlvbj5cbiAgICAgIDxlbC1vcHRpb24gdmFsdWU9XCIyXCIgbGFiZWw9XCLlhajpg6gyXCI+PC9lbC1vcHRpb24+XG4gICAgICA8ZWwtb3B0aW9uIHZhbHVlPVwiM1wiIGxhYmVsPVwi5YWo6YOoM1wiPjwvZWwtb3B0aW9uPlxuICAgICAgPGVsLW9wdGlvbiB2YWx1ZT1cIjRcIiBsYWJlbD1cIuWFqOmDqDRcIj48L2VsLW9wdGlvbj5cbiAgICAgIDxlbC1vcHRpb24gdmFsdWU9XCI1XCIgbGFiZWw9XCLlhajpg6g1XCI+PC9lbC1vcHRpb24+XG4gICAgICA8ZWwtb3B0aW9uIHZhbHVlPVwiNlwiIGxhYmVsPVwi5YWo6YOoNlwiPjwvZWwtb3B0aW9uPlxuICAgICAgPGVsLW9wdGlvbiB2YWx1ZT1cIjdcIiBsYWJlbD1cIuWFqOmDqDdcIj48L2VsLW9wdGlvbj5cbiAgICA8L2VsLXNlbGVjdD5cbiAgPC9kaXY+XG48L3RlbXBsYXRlPlxuXG48c2NyaXB0IHNldHVwIGxhbmc9XCJ0c1wiPlxuaW1wb3J0IHsgcmVmLCBjb21wdXRlZCB9IGZyb20gJ3Z1ZSdcblxuY29uc3QgbW9kZWxWYWx1ZSA9IHJlZihbXSlcbi8vIGNvZGUgaGVyZVxuPC9zY3JpcHQ+XG4iLCJpbXBvcnRfbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7fVxufSIsIl9vIjp7fX0=)

**Comment by kooriookami** [comment]
已经修复
